roles/kubernetes-apps/container_engine_accelerator/nvidia_gpu/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+nvidia_accelerator_enabled: false
+nvidia_driver_version: "384.111"
```

Comment:
390.87 was just released, not sure if that would work.

Reply:
let me test this. I think it should work

Reply:
works